docs/content/docs/testing/telemetry.md

```diff
@@ -240,24 +242,54 @@ export class ConsoleLogger implements ITelemetryBaseLogger {
 }
 ```
 
-This custom logger is then created and passed into the `getContainer()` and `createContainer()` in a `TinyliciousClient`
-object. The custom logger is now hooked up to the container's telemetry system.
+This custom logger is then created in the `azureConfig` before being passed into `AzureClient`
+constructor. The custom logger is now hooked up to the container's telemetry system.
 
 ```ts
 async function start(): Promise<void> {
-    // Create a custom ITelemetryBaseLogger object to pass into the Tinylicious container
-    // and hook to the Telemetry system
-    const consoleLogger: ConsoleLogger = new ConsoleLogger();
-
-    // Get or create the document depending if we are running through the create new flow
-    const client = useAzure ? AzureClient :  new TinyliciousClient();
-    const [fluidContainer, containerServices] = createNew
-        ? await client.createContainer({ id: containerId, logger: consoleLogger }, containerSchema)
-        : await client.getContainer({ id: containerId, logger: consoleLogger }, containerSchema);
+  // Create a custom ITelemetryBaseLogger object to pass into the Tinylicious container
+  // and hook to the Telemetry system
+  const azureConfig = {
+      connection: connectionProps,
+      logger: new ConsoleLogger(),
+  };
+  const client = new AzureClient(azureConfig);
+  let container: FluidContainer;
+  let services: AzureContainerServices;
+  let id: string;
+
+  // Get or create the document depending if we are running through the create new flow
+  const createNew = !location.hash;
+  if (createNew) {
+      // The client will create a new detached container using the schema
+      // A detached container will enable the app to modify the container before attaching it to the client
+      ({container, services} = await client.createContainer(containerSchema));
 ```
 
 Now, whenever a telemetry event is encountered, the custom `send()` method gets called and will print out the entire
 event object.
 
 <img src="https://fluidframework.blob.core.windows.net/static/images/consoleLogger_telemetry_in_action.png" alt="The
   ConsoleLogger sends telemetry events to the browser console for display.">
+
+{{% callout warning %}}
+
+The purpose of `ConsoleLogger` is purely to demonstrate how the `logger` interface should be implemented. The Fluid Framework provides `DebugLogger` by default if custom logger is not provided. For more information, please refer to [Using DebugLogger](#using-debuglogger) below instead of implementing something similar to `ConsoleLogger`.
```

Comment:
```suggestion
The purpose of `ConsoleLogger` is purely to demonstrate how the `ITelemetryBaseLogger` interface should be implemented. The Fluid Framework provides `DebugLogger` by default if a custom logger is not provided. For more information, please refer to [Using DebugLogger](#using-debuglogger) below instead of implementing something similar to `ConsoleLogger`.
```